### ISSUE TITLE
Wizards go blind after doing the ultimate dap 3 times

### ISSUE THREAD

**Issue by Contrabang** [ParadiseSS13/Paradise#25377]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

1633

### Issue Description

Wizards go blind after doing the ultimate dap 3 times

### What did you expect to happen?

not to go blind

### What happened instead?

went blind

### Why is this bad/What are the consequences?

its booooring, i want exploding wizards

### Steps to reproduce the issue.

be 2 wizards, highfive a lot, go blind

### When did the problem start happening?

prob always

**Comment by 1080pCat** [comment]
Just mirroring from discord
The `*dap` explosion wasn't what blinded you, there was a box of flashbangs that the explosion triggered that then blinded you.
God mode not protecting from that kinda stuff is.. something, but.

**Comment by Contrabang** [comment]
Yeah, wizards with godmode should still probably have some kind of resistance to this
